### ISSUE TITLE
nmol/mg{prot} and nmol/mmol{creat} two times

### ISSUE THREAD

**Issue by ahodelin** [ucum-org/ucum#266]
Hello,

The units nmol/mg{prot} and nmol/mmol{creat} apper in the files TableOfExampleUcumCodesForElectronicMessaging.xls two times.
- at Row 623 nmol/mg{prot} - nanomole per milligram of protein
- at Row 604 nmol/mg{prot} - nanomole of 1/2 cystine per milligram of protein
- at Row 603 nmol/mmol{creat} - nanomole bone collagen equivalent per millimole of creatinine
- at Row 630 nmol/mmol{creat} - nanomole per millimole of creatinine

These units appear two times in the files .pdf and .doc too but some of them at different rows.

Thanks and regards

Abel

**Comment by gschadow** [comment]
Indeed, and the text on the other side is different for each. This goes to show that the details of what was measured and how are often not carried by the units. This table was created from a list of names (which you show on the right side) and sometimes different names with such qualifiers are mapped to the same unit.

This table should _never_ be used to print display names given the UCUM unit.

------------
New status: question answered.
